src/platform/silabs/wifi/SiWx/WifiInterfaceImpl.cpp

```diff
@@ -384,7 +384,7 @@ sl_status_t SetWifiConfigurations()
     VerifyOrReturnError(status == SL_STATUS_OK, status,
                         ChipLogError(DeviceLayer, "sl_wifi_set_listen_interval failed: 0x%lx", status));
 
-    sl_wifi_advanced_client_configuration_t client_config = { .max_retry_attempts = 5 };
+    sl_wifi_advanced_client_configuration_t client_config = { .max_retry_attempts = 1 };
```

Comment:
nit. add comment here why the `max_retry_attempts` is `1` and not `0`

Reply:
Done